docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -95,9 +95,7 @@ Concepts like data sources and dashboards panels are hard to grasp and it’s a
 
 With this flow, selecting a data source has been greatly simplified, providing a clear overview of available data sources and allowing users to quickly connect to a new one when needed. Additionally, the flow now enables quick uploading of CSV files and access to built-in data sources.
 
-<!--
-- TODO:Add screenshots
--->
+{{< video-embed src="/media/docs/grafana/screen-recording-ds-picker-whats-new-10.mp4" max-width="750px" caption="Datasource picker flow" >}}
```

Comment:
@imatwawana datasource picker recording 👍

Reply:
@natellium I'll upload a new video to the same URL, replacing the previous version, so no changes here are needed :)